dojson/contrib/marc21/fields/bd1xx.py

```diff
@@ -312,5 +312,5 @@ def main_entry_uniform_title(self, key, value):
         'field_link_and_sequence_number': utils.force_list(
             value.get('8')
         ),
-        'nonfiling_characters': key[3] if key[3] else '_',
+        'nonfiling_characters': utils.int_with_default(key[3], '_'),
```

Comment:
The point is not to store `nonfiling_characters` in final data when it is not needed.